config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml

```diff
@@ -0,0 +1,66 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_paragraphs
+    - paragraphs.paragraphs_type.oe_fact
+    - paragraphs.paragraphs_type.oe_facts_figures
+  module:
+    - entity_reference_revisions
+id: paragraph.oe_facts_figures.field_oe_paragraphs
+field_name: field_oe_paragraphs
+entity_type: paragraph
+bundle: oe_facts_figures
+label: Fact
+description: 'Add between 1 to 6 Facts.'
+required: true
+translatable: true
```

Comment:
Entity reference fields referencing paragraphs should always be set to `translatable: false`. It's something the paragraphs module advocates.